src/main.rs

```diff
@@ -1,5 +1,6 @@
 extern crate structopt;
-extern crate xcb;
+extern crate x11rb;
```

Comment:
extern crate shouldnt be needed, we're using 2018 edition

Reply:
This is to do with supporting Nix

Reply:
2018 edition is fine.

Reply:
Yay lets cleanup